op-node/rollup/derive/fuzz_parsers_test.go

```diff
@@ -49,16 +49,42 @@ func FuzzL1InfoRoundTrip(f *testing.F) {
 		in := L1BlockInfo{
 			Number:         number,
 			Time:           time,
-			BaseFee:        BytesToBigInt(baseFee),
+			Basefee:        BytesToBigInt(baseFee),
 			BlockHash:      common.BytesToHash(hash),
 			SequenceNumber: seqNumber,
 		}
-		enc, err := in.MarshalBinary()
+		enc, err := in.marshalBinaryBedrock()
 		if err != nil {
 			t.Fatalf("Failed to marshal binary: %v", err)
 		}
 		var out L1BlockInfo
-		err = out.UnmarshalBinary(enc)
+		err = out.unmarshalBinaryBedrock(enc)
+		if err != nil {
+			t.Fatalf("Failed to unmarshal binary: %v", err)
+		}
+		if !cmp.Equal(in, out, cmp.Comparer(testutils.BigEqual)) {
+			t.Fatalf("The data did not round trip correctly. in: %v. out: %v", in, out)
+		}
+
+	})
+}
+
+// FuzzL1InfoEcotoneRoundTrip checks that our Ecotone encoder round trips properly
+func FuzzL1InfoEcotoneRoundTrip(f *testing.F) {
+	f.Fuzz(func(t *testing.T, number, time uint64, baseFee, hash []byte, seqNumber uint64) {
+		in := L1BlockInfo{
+			Number:         number,
+			Time:           time,
+			Basefee:        BytesToBigInt(baseFee),
+			BlockHash:      common.BytesToHash(hash),
+			SequenceNumber: seqNumber,
```

Comment:
This should set the ecotone values properly

Reply:
whoops, fixed.